project/script_VMRCM.tcl

```diff
@@ -0,0 +1,34 @@
+# Script to generate project for VMR
+#   vivado_hls -f script_VMR_CM.tcl
+#   vivado_hls -p vmrouterCM
+# WARNING: this will wipe out the original project by the same name
+
+# get some information about the executable and environment
+source env_hls.tcl
+
+# create new project (deleting any existing one of same name)
+open_project -reset vmrouterCM
+
+# source files
+set CFLAGS {-std=c++11 -I../TrackletAlgorithm}
+set_top VMRouterCMTop
+add_files ../TrackletAlgorithm/VMRouterCMTop.cc -cflags "$CFLAGS"
+add_files -tb ../TestBenches/VMRouterCM_test.cpp -cflags "$CFLAGS"
+
+open_solution "solution1"
+
+# Define FPGA, clock frequency & common HLS settings.
+source settings_hls.tcl
+
+# data files
+add_files -tb ../emData/VMRCM/tables/
+add_files -tb ../emData/VMRCM/
+
+csim_design -compiler gcc -mflags "-j8"
```

Comment:
Please remove the `-compiler` option. Apparently it isn't supported in future versions of Vitis HLS, and we want to future-proof our code when it's easy.